.github/workflows/build-binaries.yml

```diff
@@ -302,6 +306,8 @@ jobs:
             pip3 install ${{ env.PACKAGE_NAME }} --no-index --find-links dist/ --force-reinstall
             ${{ env.MODULE_NAME }} --help
       - name: "Upload wheels"
+        # Skip for `arm`, which is not supported by PyPI.
```

Comment:
In fact, `linux_armv6l` wheels are permited to upload to PyPI at the moment.

See https://github.com/pypi/warehouse/issues/3668

Reply:
Oh wow. Maturin seems to warn on this though: "Warning: No compatible platform tag found, using the linux tag instead. You won't be able to upload those wheels to PyPI."

Reply:
Yeah the warning is intentional, because there is no official manylinux/musllinux policy for armv6l and https://github.com/pypi/warehouse/issues/3668 were talking about removing the support.